pkg/compute/rust.go

```diff
@@ -231,18 +231,25 @@ func (r Rust) Verify(out io.Writer) error {
 		return newCargoUpdateRemediationErr(err, latestFastly.String())
 	}
 
-	// If fastly-sys version doesn't meet our constraint of being within the
```

Comment:
Not part of your changeset but we should update the code comment further up in this file which is stale...

```
// 2) Check that the `1.43.0` toolchain is installed
```

to

```
// 2) Check that the desired toolchain version is installed
```